autovizwidget/autovizwidget/__init__.py

```diff
@@ -1 +1 @@
-__version__ = '0.10.4'
+__version__ = '0.10.5'
```

Comment:
Please bump to 0.11.0

You can read about semantic versioning here: http://semver.org/
Since we haven't reached 1.0 yet, we are bumping minor version whenever we introduce a new feature, and bumping build version whenever we make a bug fix.